### PR TITLE
httpflow: add page

### PR DESCRIPTION
**Pull request by WhileLoop**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/httpflow.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# httpflow
+
+> A command line utility to capture and dump HTTP streams.
+> More information: https://github.com/six-ddc/httpflow.
```
Comment (navarroaxel) [inline pages/common/httpflow.md:4]:
```suggestion
> More information: <https://github.com/six-ddc/httpflow>.
```

File: pages/common/httpflow.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Capture traffic on all interfaces:
+
+`httpflow -i any`
```
Comment (navarroaxel) [inline pages/common/httpflow.md:8]:
```suggestion
`httpflow -i {{any}}`
```

File: pages/common/httpflow.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`httpflow host host httpbin.org or host baidu.com`
+
+- Use a regexp to filter requests by urls:
+
+`httpflow -u '/user/[0-9]+'`
+
+- Read packets from pcap format binary file:
+
+`httpflow -r out.cap`
```
Comment (navarroaxel) [inline pages/common/httpflow.md:20]:
```suggestion
`httpflow -r {{out.cap}}`
```

File: pages/common/httpflow.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Write the output to a directory:
+
+`httpflow -w /tmp/http`
```
Comment (navarroaxel) [inline pages/common/httpflow.md:24]:
```suggestion
`httpflow -w {{path/to/directory}}`
```

File: pages/common/httpflow.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`httpflow -i any`
+
+- Use a bpf-style capture to filter the results:
+
+`httpflow host host httpbin.org or host baidu.com`
+
+- Use a regexp to filter requests by urls:
+
+`httpflow -u '/user/[0-9]+'`
```
Comment (navarroaxel) [inline pages/common/httpflow.md:16]: We can use:
```suggestion
`httpflow -u '{{regex}}'`
```
or
```suggestion
`httpflow -u '{{/examples/[0-9]+}}'`
```

File: pages/common/httpflow.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+> A command line utility to capture and dump HTTP streams.
+> More information: https://github.com/six-ddc/httpflow.
+
+- Capture traffic on all interfaces:
+
+`httpflow -i any`
+
+- Use a bpf-style capture to filter the results:
+
+`httpflow host host httpbin.org or host baidu.com`
```
Comment (navarroaxel) [inline pages/common/httpflow.md:12]: This is the `expression`? I'm not sure about this, but the argument should be formatted like this:
```suggestion
`httpflow {{host host httpbin.org or host baidu.com}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @WhileLoop, thanks for your submission. I left some comments.

File: pages/common/httpflow.md (hunk 6 above)
Comment (sbrl) [reply pages/common/httpflow.md:12]: Duplicated `host` there at the beginning of the token syntax?

**Comment by WhileLoop** [comment]
@navarroaxel Thanks for the feedback. I believe I have addressed all the issues.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Nice one!

**Pull request merged**